src/GafferImageUI/ImageView.cpp

```diff
@@ -393,6 +393,20 @@ bool ImageView::keyPress( const GafferUI::KeyEvent &event )
 				);
 				return true;
 			}
+			if(event.key == "Home")
```

Comment:
We're inside the "check for RGBA keys" loop here. This needs moving to be outside it.